Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Prefabs/Slate.prefab

```diff
@@ -550,8 +552,8 @@ MonoBehaviour:
   panZoomSmoothing: 80
   panEventReceivers: []
   centerPoint: {fileID: 0}
-  leftPoint: {fileID: 4531382727374914902}
-  rightPoint: {fileID: 5073403802680592322}
+  leftPoint: {fileID: 0}
```

Comment:
Are these fields unneeded now?

Reply:
I left them in so this wasn't a breaking change. And, developers may want to attach gameObjects to the touch points? (Rather than just a shadow.)